sdk-tests/pom.xml

```diff
@@ -44,7 +44,7 @@
     <dependency>
       <groupId>junit</groupId>
       <artifactId>junit</artifactId>
-      <version>4.12</version>
+      <version>4.13-beta-3</version>
```

Comment:
nit: should we use beta version?

Reply:
Same version as in Azure SDK. I am fine since it is for test only.